### DISCUSSION TITLE
Skip IP address from everything and just relay.

### DISCUSSION THREAD

**Discussion by JSuenram** [rspamd/rspamd#4628] [Q&A] [answered]
Hello,

how/where can I enter an IP Address to just skip everything and just relay?
Which config-file to use, and what to "insert" there.

Regards

**Comment by fatalbanana** [comment, marked as answer]
Really everything or everything with caveats (e.g. DKIM signing / something else is still wanted)? You want `settings` module either way. To really skip **everything** you can use `want_spam` directive, or you might use `symbols_enabled` for something more surgical.

https://rspamd.com/doc/faq.html#how-to-whitelist-messages-or-skip-spam-checks-for-certain-users
https://rspamd.com/doc/faq.html#can-i-just-sign-messages-using-dkim
https://rspamd.com/doc/configuration/settings.html

~~~
# local.d/settings.conf

whitelist {
	priority = low;
	ip = "1.2.3.4";
	want_spam = yes;
}
~~~

**Comment by JSuenram** [comment]
> Really everything or everything with caveats (e.g. DKIM signing / something else is still wanted)? You want `settings` module either way. To really skip **everything** you can use `want_spam` directive, or you might use `symbols_enabled` for something more surgical.
> 
> https://rspamd.com/doc/faq.html#how-to-whitelist-messages-or-skip-spam-checks-for-certain-users https://rspamd.com/doc/faq.html#can-i-just-sign-messages-using-dkim https://rspamd.com/doc/configuration/settings.html
> 
> ```
> # local.d/settings.conf
> 
> whitelist {
> 	priority = low;
> 	ip = "1.2.3.4";
> 	want_spam = yes;
> }
> ```

Thank you for pointing the direction. Does work as expected. One more question. Will this also SKIP any "HAM" Learn-Logik for Bayes or Neural?

**Comment by fatalbanana** [comment]
> Thank you for pointing the direction. Does work as expected. One more question. Will this also SKIP any "HAM" Learn-Logik for Bayes or Neural?

Bayes autolearn (not manual learn) & neural will be skipped.

**Comment by JSuenram** [comment]
> > Thank you for pointing the direction. Does work as expected. One more question. Will this also SKIP any "HAM" Learn-Logik for Bayes or Neural?
> 
> Bayes autolearn (not manual learn) & neural will be skipped.

Thanks for reply.

Would this keep BAYES and NEURAL-Learning enabled for the 2 IPs?

`  sign_networks {
    ip = ["10.0.0.10/32", "10.0.0.11/32"];
    apply {
      symbols_enabled = ["DKIM_SIGNED","NEURAL_HAM","BAYES_HAM"]; # add ARC_SIGNED for ARC signing
      flags = ["skip_process"]; # To skip message processing
    }
`

**Reply by fatalbanana** [reply]
Neural learns based on symbols in a message & the spam/ham determination & classifies based on symbols (mostly) so it does not make good sense to enable it in this context, though it [can work with settings](https://rspamd.com/doc/modules/neural.html#settings-usage) somehow.

To enable bayes, add both of `BAYES_HAM` and `BAYES_SPAM` to `symbols_enabled`.
